core/vtk/CMakeLists.txt

```diff
@@ -1,20 +1,18 @@
 include(functions.cmake)
 
-option(TTK_FORCE_VTK_PARAVIEW "Force to use the VTK bundled into ParaView" OFF)
```

Comment:
I think you wanted to keep that one in (instead of the one in standalone/CMakeLists.txt, which is less general)